src/wf/workflow.hpp

```diff
@@ -229,10 +229,7 @@ class AMSWorkflow
 
   void set_hdcache(HDCache<FPTypeValue> *_hdcache) { hdcache = _hdcache; }
 
-  ~AMSWorkflow()
-  {
-    DBG(Workflow, "Destroying Workflow Handler");
-  }
+  ~AMSWorkflow() { DBG(Workflow, "Destroying Workflow Handler"); }
```

Comment:
is this correctly formatted?

Reply:
Yes it is. We need to add an action to always test that.